README.plugin.md

```diff
@@ -7,6 +7,18 @@
 See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
 for how to install Foreman plugins
 
+## Migrating your plugin:
```

Comment:
This is user documentation, it shouldn't say "your plugin", it isn't the user's plugin. This is also only applicable to source installations, it shouldn't be prominent in users docs.